qcodes/instrument/parameter.py

```diff
@@ -669,13 +715,19 @@ def inter_delay(self, inter_delay: Number) -> None:
 
     @property
     def full_name(self) -> str:
+        """
+        Name of the parameter including the name of the instrument and
+        submodule that the parameter may be bound to.
+        """
         return "_".join(self.name_parts)
 
     def set_validator(self, vals):
         """
-            Deprecated Set a validator `vals` for this parameter.
-                Args:
-                    vals (Validator):  validator to set
+        Deprecated Set a validator `vals` for this parameter.
```

Comment:
some punctuation for readability
```suggestion
        (Deprecated) Set a validator `vals` for this parameter.
```